docs/wiki/User Guide/REST-APIs.md

```diff
@@ -486,10 +486,14 @@ The following POST request will let Kafka Cruise Control stop an ongoing `rebala
 
 Supported parameters are:
 
-| PARAMETER | TYPE      | DESCRIPTION                                   | DEFAULT   | OPTIONAL  |
-|-----------|-----------|-----------------------------------------------|-----------|-----------|
-| json      | boolean   | return in JSON format or not                  | false     | yes       | 
-| doAs      | string    | propagated user by the trusted proxy service  | null      | yes       | 
+| PARAMETER           | TYPE      | DESCRIPTION                                                                                                   | DEFAULT   | OPTIONAL  |
+|---------------------|-----------|---------------------------------------------------------------------------------------------------------------|-----------|-----------|
+| force_stop          | boolean   | (not supporeted in Kafka 2.4 or above) stop an ongoing execution forcefully by deleting Kafka internal zNodes | false     | yes       |
+| stop_external_agent | boolean   | (required Kakfa 2.4 or above) stop an ongoing execution even if it is started by an external agent            | true      | yes       |
```

Comment:
Typo: `Kakfa` -> `Kafka`

Reply:
Just a suggestion -- we may also state this in terms of CC versions rather than Kafka versions. For example, maybe we can say something like: "This parameter is a no-op for CC versions `2.3.*`."